doc/ru/qbittorrent-nox.1.md

```diff
@@ -0,0 +1,2 @@
+# НАЗВАНИЕ
```

Comment:
You should at least add the title block to this file (and to doc/ru/qbittorrent.1.md too) :
https://github.com/qbittorrent/qBittorrent/blob/e045b4678d28fad93625e70d45a13009649a9d33/doc/qbittorrent.1.md?plain=1#L1-L3

https://pandoc.org/MANUAL.html#extension-pandoc_title_block

Reply:
Done.